### ISSUE TITLE
Signup: Add two Starter Designs

### ISSUE THREAD

**Issue by ianstewart** [Automattic/wp-calypso#48381]
We've created two new Starter Design sites that can be added to the Design Screen in signup.

1. https://pollardstarter.wordpress.com/
2. https://balanstarter.wordpress.com/

There are a few themes issues surfaced by these Starter Designs that would be good to get closed before these are added to the design screen in signup.

- https://github.com/Automattic/themes/issues/2896
- https://github.com/Automattic/themes/issues/2895
- https://github.com/Automattic/themes/issues/2918

Additionally, as these are "short" designs, they'll look their best with slightly shorter thumbnails. Something to consider for further improvements to this screen. cc @joanrho @rickybanister 

Speaking of, here's what they look like right now (with those issues present) …

![image](https://user-images.githubusercontent.com/545779/102273537-188a8c80-3ee8-11eb-9055-0cc9d5c8d201.png)

cc @iamtakashi @alaczek 

### Recipe

The starter design "recipe" for both sites looks like the following.

- [Mayland Theme](https://maylanddemo.wordpress.com/).
- Reduce default content to one static homepage.
- Update block-based content in the homepage. (Expect the content here, and similar content to show up in the Editor as Patterns.)
- Update global fonts in the Editor.
- Set custom colour scheme and remove menu in the customizer.

These sites will not receive a blog page by default but just like a custom menu with additional pages that will be quite easy to add later.

We'll want to continue to make improvements to these Starter Designs as block-based themes are added to Signup.

**Comment by andrewserong** [comment]
@ianstewart I took a look at adding these designs in #48727. I think it's currently blocked by the following two main issues:

1. The colour styles bleeding into the editor, which I believe are already covered in the linked issues in the themes repo.
2. The Gutenboarding layout for the design picker doesn't play nicely with short screenshots for these designs.

| Balan | Pollard |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/103978924-b7956200-51d0-11eb-9568-928d575e175b.png) | ![image](https://user-images.githubusercontent.com/14988353/103978888-a77d8280-51d0-11eb-8977-a535bb112438.png) |

As an aside, I _love_ the look of these designs, though!

**Comment by p-jackson** [comment]
> As an aside, I love the look of these designs, though!

Same! This seems like a really useful scenario to cover in our starter designs!

**Comment by rickybanister** [comment]
Can the default width of the browser be adjusted when snapping the mshot? These are intended to be 'mobile-first' style designs and should be zoomed into the content more vs short as you show above.

**Comment by p-jackson** [comment]
> Can the default width of the browser be adjusted when snapping the mshot?

We can pick the browser width yes. I wonder if in that case we should also auto-select the mobile view when advancing to the next step.

<img width="432" alt="Screenshot 2021-01-13 at 9 59 04 AM" src="https://user-images.githubusercontent.com/1500769/104374102-3d752c80-5586-11eb-810f-46dda5b4a440.png">

That way the user isn't surprised that it looks different from what they saw in the gallery. Plus they might get a subtle clue that the layout they picked might look different on desktop.

**Comment by ianstewart** [comment]
cc @ollierozdarz

**Comment by andrewserong** [comment]
> Can the default width of the browser be adjusted when snapping the mshot? These are intended to be 'mobile-first' style designs and should be zoomed into the content more vs short as you show above.

That's a good point, and like Phil mentions we can set a different browser width for each screenshot if we need to. The only thing is, how would those screenshots look next to each other if some are optimised for desktop and others for mobile. I could imagine it looking a bit odd if some are more zoomed in than others.

We can definitely play around with it. If we can get a mock-up for how it should look in the Design step, that'll help us get started. I'm curious if we'll need or want more of a masonry layout like in p9Jlb4-22o-p2 ?

**Comment by ianstewart** [comment]
Note: I've made an update to these two starter designs to use the new option in Varia to hide the footer and header on the homepage. No blockers and it should "just work" but if the output doesn't match the starter design because of a difference there — it doesn't "just work." :)

**Comment by lsl** [comment]
@ianstewart #48727 is ready for deploy as soon as you're ready with the remaining two issues:

- https://github.com/Automattic/themes/issues/2895
- https://github.com/Automattic/themes/issues/2918

**Comment by ianstewart** [comment]
2918 is closed so we're just down to 2895 now.

**Comment by simison** [comment]
2895 is closed (https://github.com/Automattic/themes/issues/2895#issuecomment-774133421) and will ship next week 🎉
